vignettes/multiple-cities.Rmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "5. Multiple delineations"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{5. Multiple delineations}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(CRiSp)
+```
+
+## Motivation
+
+Comparative analysis can shed light on patterns that are not visible in a single case. The automated delineation provided by CRiSp allows for a reliable frame of reference for such comparisons, including:
+
+1. all cities along a large river;
+1. all river-crossed cities of a given size;
+1. all cities crossed by rivers of the same kind in a given region.
```

Comment:
The numbers need a fix, now they are all `1.`

Reply:
That's intentional, will show up correctly when rendered. I was doing that while compiling the list and changing their order, but once it is final I can also number them properly in the source document.